### PR TITLE
Mobile view: dnd beyond search button fixed

### PR DESCRIPTION
**Pull request by BillSheikh1**

### CODE REVIEW

**Comment by BillSheikh1** [comment]
In mobile view, the menu dropdown container overlapped with the search button. Changed styles to remove it

File: src/components/App.css [hunk 1]
```diff
@@ -175,6 +175,16 @@ input[type="number"] {
 }
 
 .battle-toolbar--button {
+  padding: 4px;
```
Comment (Paul-Ladyman) [inline src/components/App.css:178]: Really just need two battle toolbar button classes:

```
// For all buttons
.battle-toolbar--button {
  padding: 4px;
  height: 40px;
  margin-top-5px;
}

// modifier for dropdown buttons
.battle-toolbar--button__dropdown {
  margin-top: 25px;
}
```

File: src/components/App.css [hunk 2]
```diff
@@ -175,6 +175,16 @@ input[type="number"] {
...
+  height: 40px;
+  margin-top: 25px;
```
Comment (Paul-Ladyman) [inline src/components/App.css:180]: Previously the spacing of the dropdown items was provided by these rules on `battle-toolbar--options-dropdown`:

```
display: flex;
justify-content: space-between;
flex-direction: column;
```

It looks like if we go with a larger `margin-top` instead the above rules are no longer necessary (I removed them and it looked OK!) I think the `width` of `.battle-toolbar--options-container` plus the `margin-top` gets them to display vertically. If this seems good we could remove these extra styles to simplify things a bit.

File: src/components/App.css [hunk 3]
```diff
@@ -175,6 +175,16 @@ input[type="number"] {
...
+}
+
+.battle-toolbar--button__top, .battle-toolbar--button__progress {
+  margin-top: 5px !important;
```
Comment (Paul-Ladyman) [inline src/components/App.css:184]: Avoid using `!important` if at all possible. I removed it in the browser and it doesn't seem to be necessary.

File: src/components/App.css [hunk 4]
```diff
@@ -199,8 +209,8 @@ input[type="number"] {
   display: flex;
   flex-direction: column;
   align-items: center;
-  width: 40px;
-  height: 314px;
+  width: 40px;
```
Comment (Paul-Ladyman) [inline src/components/App.css:212]: Remove whitespace.

File: src/components/App.css [hunk 5]
```diff
@@ -216,7 +226,7 @@ input[type="number"] {
   justify-content: space-between;
   flex-direction: column;
   margin-top: 10px;
-  height: 100%;
+  height: 314px;
```
Comment (Paul-Ladyman) [inline src/components/App.css:229]: The height on `.battle-toolbar--options-dropdown` and `.battle-toolbar--options-container` do not seem to be necessary. The dropdown takes whatever height it needs to contain the buttons.

File: src/components/BattleToolbar.js [hunk 6]
```diff
@@ -70,6 +70,7 @@ class BattleToolbar extends Component {
     const nextButtonTitle = round === 0 ? 'Start battle' : 'Next initiative';
     const optionsMenuIcon = this.state.optionsExpanded ? <OptionsMenuOpenIcon /> : <OptionsMenuClosedIcon />;
     const optionsClass = this.state.optionsExpanded ? 'battle-toolbar--options-dropdown' : 'hidden';
+    // const toggleButtonClass = this.state.optionsExpanded ? 'battle-toolbar--button__toggle--close' : 'battle-toolbar--button__toggle--open';
```
Comment (Paul-Ladyman) [inline src/components/BattleToolbar.js:73]: Remove this comment.

File: src/components/BattleToolbar.js [hunk 7]
```diff
@@ -120,14 +121,14 @@ class BattleToolbar extends Component {
           <div className="battle-toolbar--options-container">
             <button
               title="Options Menu"
-              className={`${buttonClass} battle-toolbar--button__options`}
+              className={`battle-toolbar--button__toggle battle-toolbar--button__options`}
```
Comment (Paul-Ladyman) [inline src/components/BattleToolbar.js:124]: The `battle-toolbar--button__options` here wasn't actually being used in the CSS so could be removed.

**Comment by Paul-Ladyman** [comment]
Took care of the review changes in https://github.com/Paul-Ladyman/dnd-battle-tracker/pull/58.

**Pull request closed without merging**